appengine-standard-java8/cloudsql/src/main/java/com/example/appengine/cloudsql/ListResults.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.appengine.cloudsql;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@SuppressWarnings("serial")
+@WebServlet(name = "list", description = "List the latest news posts", urlPatterns = "/")
+public class ListResults extends HttpServlet {
+  // Displays the stored blog posts
+
+  Connection conn; // DB Connection object
+
+  // Table creation queries
+  final String createContentTableSql =
+      "CREATE TABLE IF NOT EXISTS posts ( post_id INT NOT NULL "
+          + "AUTO_INCREMENT, author_id INT NOT NULL, timestamp DATETIME NOT NULL, "
+          + "title VARCHAR(256) NOT NULL, "
+          + "body VARCHAR(1337) NOT NULL, PRIMARY KEY (post_id) )";
+
+  final String createUserTableSql =
+      "CREATE TABLE IF NOT EXISTS users ( user_id INT NOT NULL "
+          + "AUTO_INCREMENT, user_fullname VARCHAR(64) NOT NULL, "
+          + "PRIMARY KEY (user_id) )";
+
+  // Create a test user
+  final String createTestUserSql = "INSERT INTO users (user_fullname) VALUES ('Test User')";
+
+  // Blog post retrieval queries
+  final String selectSql =
+      "SELECT posts.post_id, users.user_fullname, posts.timestamp, posts.title, posts.body FROM posts, users "
+          + "WHERE (posts.author_id = users.user_id) AND (posts.body != \"\") ORDER BY posts.post_id DESC";
+
+  // Preformatted HTML
+  String headers =
```

Comment:
both lines below appear to be too long, you should be able to break the lines up and use concatenation of string constants